### PR TITLE
Cookiecutting - pass tests

### PR DESCRIPTION
**Pull request by 8bitsam**
Add back old files to make tests pass. Note: does not use pytest.
See screenshot. Do we want to fix deprecation warnings yet?
Screenshot from 2024-07-22 17-11-25

### CODE REVIEW

File: src/diffpy/__init__.py [hunk 1]
```diff
@@ -1,23 +1,28 @@
 #!/usr/bin/env python
```
Comment (sbillinge) [inline src/diffpy/__init__.py:1]: please change this file back to the cookiecutter version

File: src/diffpy/pdffit2/__init__.py [hunk 2]
```diff
@@ -1,24 +1,34 @@
...
-# https://github.com/diffpy/diffpy.pdffit2/graphs/contributors
-#
-# See LICENSE.rst for license information.
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
 #
 ##############################################################################
 
-"""PDFfit2 - real space structure refinement program."""
+"""PDFfit2 - real space structure refinement program.
```
Comment (sbillinge) [inline src/diffpy/pdffit2/__init__.py:16]: please change the header of this file back to the cookiecutter version.

File: src/diffpy/pdffit2/tests/debug.py [hunk 3]
```diff
@@ -1,19 +1,20 @@
 #!/usr/bin/env python
```
Comment (sbillinge) [inline src/diffpy/pdffit2/tests/debug.py:1]: we should be able to move this one back to the cookiecutter version too.

File: src/diffpy/pdffit2/tests/run.py [hunk 4]
```diff
@@ -1,34 +1,38 @@
...
-# https://github.com/diffpy/diffpy.pdffit2/graphs/contributors
-#
-# See LICENSE.rst for license information.
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
 #
 ##############################################################################
+
 """Convenience module for executing all unit tests with
+
```
Comment (sbillinge) [inline src/diffpy/pdffit2/tests/run.py:17]: move header back to cookiecutter version

File: src/diffpy/pdffit2/tests/run.py [hunk 5]
```diff
@@ -1,34 +1,38 @@
...
 if __name__ == "__main__":
-    # show output results from every test function
-    args = ["-v"]
-    # show the message output for skipped and expected failure tests
-    if len(sys.argv) > 1:
-        args.extend(sys.argv[1:])
-    print("pytest arguments: {}".format(args))
-    # call pytest and exit with the return code from pytest
-    exit_res = pytest.main(args)
-    sys.exit(exit_res)
```
Comment (sbillinge) [inline src/diffpy/pdffit2/tests/run.py:32]: keep the cookiecutter code but comment it out.  Later we want to migrate over to the cookiecutter way of running the tests if we can.

**Review by sbillinge — COMMENTED** [review]
please see my inline comments

**Comment by sbillinge** [comment]
@8bitsam shall I close this and we redo?   I think I would be more comfortable with that now that we have the pip -e install working and can work to get tests passing locally with that?

When we merge files, we need to keep old copyright statements (we don't have the legal right to remove them) but we want to move to the new cookiecutter syntax in that header region with current copyright to Columbia.

Then let's do a careful merge of code where we would like to change things towards the cookiecutter, but leave anything from Pavol that we need for backwards compatibility.

**Comment by 8bitsam** [comment]
I think that's a good idea, the commit for this is pretty straightforward, anyways.

**Pull request closed without merging**